.travis.yml

```diff
@@ -39,18 +45,18 @@ before_script:
       export DISPLAY=:99.0
     fi
   # Build budle.js
-  - npm run build
+  - yarn build
   - cp test/setup.js.travis test/setup.js
   # Start https server (and sleep) so we can get the local bundle.js via https.
   # The `&` at the end tells this process to fork into the background
   # https://github.com/travis-ci/travis-ci/issues/1321
-  - npm run live-serve-build &
+  - yarn live-serve-build &
   # Increase the shared memory mountpoint. Chrome or something in the tests is
   # taking up more shared memory than is available which causes tests to fail.
   # See https://github.com/SeleniumHQ/docker-selenium/issues/431
   - sudo mount -o remount,size=64G /run/shm
 sudo: required
```

Comment:
You can also remove this line, since container-based builds have been deprecated. 
I've already opened a pr for this in the main repo, but forgot to check the rest.